AmplifyPlugins/Predictions/AWSPredictionsPluginTests/Mocks/Service/MockTextractBehavior.swift

```diff
@@ -17,18 +17,18 @@ class MockTextractBehavior: AWSTextractBehavior {
     var error: Error?
 
     func analyzeDocument(request: AWSTextractAnalyzeDocumentRequest) -> AWSTask<AWSTextractAnalyzeDocumentResponse> {
-        if let finalResult = analyzeDocument {
-            return AWSTask(result: finalResult)
+        guard let finalError = error else {
+            return AWSTask(result: analyzeDocument)
         }
-        return AWSTask(error: error!)
```

Comment:
force unwrap of error should be avoided. I changed it to something like: https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyPlugins/Predictions/AWSPredictionsPluginTests/Mocks/Service/MockComprehendBehavior.swift#L21-L27